### PR TITLE
Removed the limitations in considering mail ids and reduced the time complexity

### PR DESCRIPTION
**Pull request by sudipg4112001**
Fixes #129

`if not re.match(r'[^@]+@[^@]+\.[^@]+', email):` this line is limited only to some mail ids. So, I have tried to resolve this issue by replacing this line by `if not re.match(r'^[a-z0-9]+[\._]?[a-z0-9]+[@]\w+[.]\w+$', email):` I guess this will resolve the issue and all types of mail ids will be considered.
Secondly, 
` continue`
`ask_email = False`
 I feel these lines can be omitted. After, omission it will reduce the time complexity without affecting the main process.

### CODE REVIEW

**Comment by iamareebjamal** [comment]
Making it O(∞) reduces complexity?

**Review by sudipg4112001 — COMMENTED** [review]
Other than checking only "@" and "." in a mail, I feel we should also check alphabets, numbers, alphanumeric characters.

File: create_db.py [hunk 1]
```diff
@@ -16,10 +16,8 @@ def create_default_user(email, password):
         ask_email = True
         while ask_email:
             email = input("Enter email for super_admin    : ")
-            if not re.match(r'[^@]+@[^@]+\.[^@]+', email):
+            if not re.match(r'^[a-z0-9]+[\._]?[a-z0-9]+[@]\w+[.]\w+$', email):
                 print('\nInvalid email address\n')
-                continue
```
Comment (sudipg4112001) [inline create_db.py:21]: For what purpose is the `continue` statement used ?

File: create_db.py [hunk 2]
```diff
@@ -16,10 +16,8 @@ def create_default_user(email, password):
         ask_email = True
         while ask_email:
             email = input("Enter email for super_admin    : ")
-            if not re.match(r'[^@]+@[^@]+\.[^@]+', email):
+            if not re.match(r'^[a-z0-9]+[\._]?[a-z0-9]+[@]\w+[.]\w+$', email):
```
Comment (sudipg4112001) [inline create_db.py:19]: It is checking only "@" and "." rather it should check alphanumeric characters also, to bring more accuracy in checking the email ids.

File: create_db.py [hunk 3]
```diff
@@ -16,10 +16,8 @@ def create_default_user(email, password):
...
                 print('\nInvalid email address\n')
-                continue
-            ask_email = False
```
Comment (sudipg4112001) [inline create_db.py:22]: What is the use of this statement ?

**Pull request closed without merging**